services/tests/test_services_graphql_api.py

```diff
@@ -104,6 +103,5 @@ def test_normal_user_cannot_add_service_multiple_times_mutation(rf, user_gql_cli
     # do the mutation again
     executed = user_gql_client.execute(query, context=request)
     assert "errors" in executed
-    assert executed["errors"][0]["message"] == _(
-        "Service already exists for this profile!"
-    )
+    assert "code" in executed["errors"][0]["extensions"]
+    assert executed["errors"][0]["extensions"]["code"] == SERVICE_ALREADY_EXISTS_ERROR
```

Comment:
As these checks are probably required more in the future as well, one could consider adding a helper for this, something like `assert_error(executed, SERVICE_ALREADY_EXISTS_ERROR)`

Reply:
True that. Since there are lots of them I'll write a ticket about this.